src/DroneSafetyApi/Models/IDataPointRepository.cs

```diff
@@ -6,10 +6,10 @@
 
 namespace DroneSafetyApi.Models
 {
-    public interface IHazardsRepository
+    public interface IDataPointRepository
```

Comment:
DataPoint should be in the plural. This change should be propagated to all relevant places :)